lib/chef/application/solo.rb

```diff
@@ -241,6 +241,13 @@ def reconfigure
         ARGV[dash_r] = "--recipe-url"
       end
 
+      # For back compat reasons, we need to ensure that we try and use the cache_path as a repo first
+      Chef::Log.debug "Current chef_repo_path is #{Chef::Config.chef_repo_path}"
```

Comment:
this seems likely to confuse someone if it says the path is a certain thing and then immediately switches to a different path?